qiita_db/search.py

```diff
@@ -310,7 +310,7 @@ def _parse_study_search_string(self, searchstr,
                     meta_header_type_lookup[header] = 'varchar'
 
         # create the study finding SQL
-        # remove metadata headers that are in required_sample_info table
+        # remove metadata headers that are in study_sample table
         meta_headers = tuple(meta_headers.difference(
```

Comment:
I think you can just directly drop the sample column now if needed, otherwise this is not required